
void test() {
    continue; // expected-error {{'continue' statement not in loop statement}}
}
